frontend/src/components/pages/recommandationForm/RecommendationForm.js

```diff
@@ -0,0 +1,74 @@
+import styled from "styled-components/macro"
+
+import { useState } from "react"
+import Summary from "./stepcomponents/Summary"
+import {useHistory} from "react-router";
+import Step from "./stepcomponents/Step";
+import QuestionData from "./QuestionData.json"
+
+
+function RecommendationForm() {
+   const [questionData,setQuestionData]=useState(QuestionData[0])
+   const [wineConfiguration, setWineConfiguration] = useState({
+      occasion: "",
+      wineStyle: "",
+      region: "",
+      alcohol: "",
+      taste: "",
+   })
+
+   const history=useHistory();
+
+   const nextStep = (value) => {
+      if(parseInt(value)!==7){
+         setQuestionData(QuestionData[parseInt(value)])
+      }else{
+         setQuestionData("finished")
+      }
+
+   }
+   const handleDataInput = (name, value) => {
+      console.log("Received "+name, value)
+      setWineConfiguration({ ...wineConfiguration, [name]: value })
+   }
+
+   const handleSubmit = () => {
+      //future PostMapping
+      console.log("Data send")
+      history.push("/recommendations")
+   }
+
+   return (
+      <FormWrapper>
+         <MainHeading>Let's find your wine</MainHeading>
+         <Form>
+            { questionData!=="finished"?
+               <Step toAskData={questionData} handleDataInput={handleDataInput} nextStep={nextStep}/>:
```

Comment:
I would stick with `questionData` or`question`

```suggestion
               <Step question={questionData} handleDataInput={handleDataInput} nextStep={nextStep}/>:
```